workflows/general/samtools-merge.wdl

```diff
@@ -20,7 +20,7 @@ workflow samtools_merge {
     }
     input {
         Array[File] bams
-        String prefix = basename(bams[0], ".bam")  # TODO is this a sane default? Or should be required?
```

Comment:
Did you check where this is used to ensure that we're providing something?

Reply:
Yea we're good on that front. Only used in `dnaseq-core` and there `prefix` is supplied.